app/views/feedback.jade

```diff
@@ -7,7 +7,28 @@ html
     meta(name='viewport' content='width=device-width, initial-scale=1')
 
     link(rel='stylesheet' href='styles/main.css')
-    link(rel='script' href='scripts/satisfaction.js')
+    script(type="text/javascript" src="https://l2.io/ip.js?var=myip")
+    script.
+      window.onload = function () {
+        var satisfaction = getQueryVariable('satisfaction');
+        var origin = getQueryVariable('origin');
+        document.getElementById('result').value = satisfaction;
+        document.getElementById('origin').value = origin;
+        document.getElementById('ip').value = myip;
+        document.getElementById('satisfaction').submit();
+      }
+      function getQueryVariable(variable) {
+        var query = window.location.search.substring(1);
+        var vars = query.split('&');
```

Comment:
why not just chain .split onto the substring and rename query to vars?
the query variable isn't used ever again